dashboard/src/components/SideMenu/SendFeedback.tsx

```diff
@@ -0,0 +1,74 @@
+import { MdOutlineFeedback } from 'react-icons/md';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from '@/components/ui/alert-dialog';
+
+import {
+  FEEDBACK_ISSUE_URL,
+  FEEDBACK_EMAIL_TO,
+} from '@/utils/constants/general';
+
+import { NavigationMenuItem } from '@/components/ui/navigation-menu';
+
+import NavLink from './NavLink';
+
+const get_encoded_url = (): string => encodeURIComponent(window.location.href);
```

Comment:
you propably sould use tanstack router here, I think it is `router.location` from `useRouter` 
https://tanstack.com/router/latest/docs/framework/react/api/router/RouterStateType

Reply:
I think this will just make us `listen` for router state changes, but we don't need to do that here.